CONTRIBUTING.md

```diff
@@ -13,13 +31,26 @@ please head over to our [feedback page](https://feedback.mongodb.com/forums/9308
 Please create a [GitHub issue](https://github.com/mongodb/mongocli/issues/new?assignees=&labels=&template=bug_report.md) describing the kind of problem you're facing
 with as much detail as possible, including things like operating system or anything else may be relevant to the issue.
 
-## Submitting a Patch
+## Submitting Patches
 
-Before submitting a patch to the repo please consider opening an [issue first](#reporting-issues).
+The MongoDB CLI project welcomes all contributors and contributions regardless of skill or experience level. 
+If you are interested in helping with the project, we will help you with your contribution.
 
-### Contributor License Agreement
+### Code Contribution Guidelines
 
-For patches to be accepted, contributors must sign our [CLA](https://www.mongodb.com/legal/contributor-agreement).
+We want to create the best possible product for our users and the best contribution experience for our developers,
+we have a set of guidelines to ensure that all contributions are acceptable.
+
+To make the contribution process as seamless as possible, we ask for the following:
+
+* Go ahead and fork the project and make your changes.  
+  We encourage pull requests to allow for review and discussion of code changes.
+* When you’re ready to create a pull request, be sure to:
+  * Sign the [CLA](https://www.mongodb.com/legal/contributor-agreement).
+  * Have test cases for the new code. If you have questions about how to do this, please ask in your pull request.
```

Comment:
[nit] I'd link to `Generating Mocks` at this point to help users write well-structured tests.